src/main/java/org/redhat/mdcr/config/PropertyConfiguration.java

```diff
@@ -0,0 +1,15 @@
+package org.redhat.mdcr.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+public class PropertyConfiguration {
+
+	@Configuration
+	@PropertySource({ "file:/deployments/config/configmap.properties" })
```

Comment:
@vbkgithub can you help me understand this a little? I think we want to stay with the default Spring Boot application config source.